comp/netflow/server/server.go

```diff
@@ -123,3 +153,50 @@ func (s *Server) Stop() {
 	}
 	s.running = false
 }
+
+// IsEnabled checks if the netflow functionality is enabled in the configuration.
+func IsEnabled() bool {
+	return config.Datadog.GetBool("network_devices.netflow.enabled")
+}
+
+// GetStatus retrieves the current status of the server with details about
+// all listeners and categorizes them into working and closed.
+func GetStatus() NetflowServerStatus {
+	globalServerMu.Lock()
+	defer globalServerMu.Unlock()
+
+	if globalServer == nil {
+		return NetflowServerStatus{
+			TotalListeners:         0,
+			OpenListeners:          0,
+			ClosedListeners:        0,
+			WorkingListenerDetails: []NetflowListenerStatus{},
+			ClosedListenerDetails:  []NetflowListenerStatus{},
+		}
+	}
+
+	workingListeners := []NetflowListenerStatus{}
+	closedListenersList := []NetflowListenerStatus{}
+
+	for _, listener := range globalServer.listeners {
+		if listener.Error != nil {
+			closedListenersList = append(closedListenersList, NetflowListenerStatus{
+				Config: listener.config,
+				Error:  listener.Error.Error(),
+			})
+		} else {
+			workingListeners = append(workingListeners, NetflowListenerStatus{
+				Config: listener.config,
+				Error:  "No Error(s) Found",
```

Comment:
```suggestion
```
Seems weird to me to say "Error= No Error(s) Found". Can we just leave that nil?